kitsune/dashboards/readouts.py

```diff
@@ -292,6 +292,16 @@ def single_result(sql, params):
 
     if product:
         total = total.filter(products=product)
+        has_forum = product.questions_locales.filter(locale=locale).exists()
+
+    ignore_categories = [str(ADMINISTRATION_CATEGORY),
+                         str(NAVIGATION_CATEGORY),
+                         str(HOW_TO_CONTRIBUTE_CATEGORY)]
+
+    if not product or not has_forum:
+        ignore_categories.append(str(CANNED_RESPONSES_CATEGORY))
+
+    total = total.exclude(category__in=ignore_categories)
 
     total_docs = total.filter(is_template=False).exclude(
         category__in=[HOW_TO_CONTRIBUTE_CATEGORY]).count()
```

Comment:
^^ This guy is excluding one of the categories you already excluded above. I think you can remove it here?